minification.ipynb

```diff
@@ -0,0 +1,993 @@
+{
```

Comment:
<p>Ideally this can be run from colab, and this step would prevent this. Feel free to upload this model somewhere (zenodo)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scvi-tutorials/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>it can still be run from colab, I just need to uncomment this part (like we do in the other tutorials basically..)</p>